notebooks/Watson_NLU_Demo.ipynb

```diff
@@ -17823,7 +17823,9 @@
     }
    ],
    "source": [
-    "# TODO: Use a Gremlin query to find all the children of this LCA\n",
+    "# TODO: Replace with something equivalent \n",
```

Comment:
Gremlin functionality is commented out in this notebook, will replace it with something equivalent in a followup

Reply:
Sounds good.